src/react-native/04-intro-to-jsx/intro-to-jsx.md

```diff
@@ -39,10 +43,47 @@ If you’ve implemented the solution correctly, the tests will pass when you run
 <details>
 <summary>Click to see the solution</summary>
 
+✏️ Update **src/App.tsx** to the following:
+@diff ../../../exercises/react-native/03-creating-a-new-app/04-solution/App.tsx ../../../exercises/react-native/04-intro-to-jsx/01-solution/App.tsx only
+
+</details>
+
+## Objective 2: Conditions and loops
+
+TODO
+
+### Concept TODO
+
+TODO
+
+### Setup 2
+
+✏️ Update **src/App.tsx** to the following:
+
+@sourceref ../../../exercises/react-native/04-intro-to-jsx/01-solution/App.tsx
+
+### Verify 2
+
+✏️ Update **src/App.test.tsx** to the following:
+
+@diff ../../../exercises/react-native/04-intro-to-jsx/01-solution/App.test.tsx ../../../exercises/react-native/04-intro-to-jsx/02-solution/App.test.tsx only
+
+### Exercise 2
+
 TODO
 
+### Solution 2
+
+If you’ve implemented the solution correctly, the tests will pass when you run `npm run test`!
+
+<details>
+<summary>Click to see the solution</summary>
+
+✏️ Update **src/App.tsx** to the following:
+@diff ../../../exercises/react-native/04-intro-to-jsx/01-solution/App.tsx ../../../exercises/react-native/04-intro-to-jsx/02-solution/App.tsx only
+
 </details>
 
 ## Next steps
 
-TODO
\ No newline at end of file
+Next is [learn-react-native/intro-to-testing].
```

Comment:
This should encourage the learner to go to the next page, e.g. `Next we will learn the [basics of testing](./intro-to-testing.html) React Native applications.`